pom.xml

```diff
@@ -65,7 +65,7 @@
     <logstash-logback.version>4.5.1</logstash-logback.version>
     <metrics-spring.version>3.0.0</metrics-spring.version>
     <nosqlunit.version>0.7.9</nosqlunit.version>
-    <obiba-commons.version>1.8.4</obiba-commons.version>
+    <obiba-commons.version>1.8.5</obiba-commons.version>
```

Comment:
There is no difference between 1.8.x and master.
I don't know why these 2 branches are separated ...